baseplate/integration/thrift/__init__.py

```diff
@@ -20,6 +20,7 @@ def make_processor(app_config):
 
 import sys
 
+from requests.structures import CaseInsensitiveDict
```

Comment:
We depend on requests already for installation, so this seemed like a reasonable module to include. We could also store our own implementation if desired.

Reply:
yup, that's fine. this is one of the hard deps in `setup.py`.